kalite/topic_tools/__init__.py

```diff
@@ -266,16 +287,22 @@ def get_content_cache(force=False, annotate=False, language=None):
 
     if CONTENT is None:
         CONTENT = {}
+
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP and not force:
```

Comment:
Same situation as in https://github.com/learningequality/ka-lite/pull/4122#discussion_r35056371.